src/lamplib/src/genny/templates/documentation.md.j2

```diff
@@ -1,6 +1,6 @@
-# Workload Documenation
+# Workload Documentation 
 {% for workload in workloads %}
```

Comment:
Can we add a line break inside this loop, either at the top or bottom? I think 
```
---
```
Would be sufficient.

Reply:
Ah, yeah -- the heading including the line does make that confusing.